partitioned-table.md

```diff
@@ -560,6 +560,15 @@ MOD(YEAR('2005-09-01'),4)
 =  1
 ```
 
+#### LINEAR HASH handling
+
+Before v6.4, if you execute DDL statements of MySQL Linear Hash partitions in TiDB, TiDB can only create non-partitioned tables. In this case, if you still want to use partitioned tables in TiDB, you need to update your DDL statements.
+
+Since v6.4.0, TiDB supports parsing the MySQL `PARTITION BY LINEAR HASH` syntax but ignores the `LINEAR` keyword in it. If you have some existing DDL and DML statements of MySQL Linear Hash partitions, you can execute them in TiDB without modification:
+
+- For DDL statements of MySQL Linear Hash partitions, TiDB will create non-linear Hash partitioned tables correspondingly (note that Linear Hash partitioned tables are not provided in TiDB). 
+- DML statements of MySQL Linear Hash partitions are not affected unless you use explicit partition selection, in which case TiDB will return the query results of the corresponding TiDB Hash partitioned normally.
```

Comment:
```suggestion
- For DML statements of MySQL Linear Hash partitions, as long as you do not use explicit partition selection, TiDB will return the query results of the corresponding TiDB Hash partitioned normally.
```

Reply:
@mjonss I made a new suggested change as above. PTAL. Thanks.

Reply:
@qiancai explicit partition selection, seems like it is just mentioned as 'partition selection' https://docs.pingcap.com/tidb/dev/partitioned-table#partition-selection

Reply:
@mjonss Got it. Thanks for the explanation. If so, we could add a link to partition selection as in the my suggestion below.